### ISSUE TITLE
Always open in container spawns two tabs instead of one

### ISSUE THREAD

**Issue by ph0enix49** [mozilla/multi-account-containers#728]
Some websites which are configured to "always open in <some_container>", spawn two tabs instead of one. e.g youtube.com (signed in).
Repro:
1) Create a new default tab (ctrl+t)
2) Type "youtube.com" into address bar and hit enter
3) Current tab converts to Personal and simultaneously a new Personal container tab opens with youtube.com.

If you have clicked a youtube link before - it will spawn two Personal container youtube tabs and both will start playing the video.

I'm signed in in default and personal container tabs to google. 

Raw data:

{
  "application": {
    "name": "Firefox",
    "osVersion": "Linux 4.11.11-300.fc26.x86_64",
    "version": "54.0.1",
    "buildID": "20170725113540",
    "userAgent": "Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0",
    "safeMode": false,
    "supportURL": "https://support.mozilla.org/1/firefox/54.0.1/Linux/en-US/",
    "numTotalWindows": 1,
    "numRemoteWindows": 1,
    "remoteAutoStart": true,
    "autoStartStatus": 1,
    "keyGoogleFound": false,
    "keyMozillaFound": true
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "browser.cache.disk.hashstats_reported": 1,
    "browser.cache.frecency_experiment": 1,
    "browser.cache.disk.smart_size.first_run": false,
    "browser.cache.disk.smart_size.use_old_max": false,
    "browser.cache.disk.capacity": 358400,
    "browser.cache.disk.filesystem_reported": 1,
    "browser.download.importedFromSqlite": true,
    "browser.places.smartBookmarksVersion": 8,
    "browser.sessionstore.upgradeBackup.latestBuildID": "20170725113540",
    "browser.startup.homepage": "about:blank",
    "browser.startup.homepage_override.buildID": "20170725113540",
    "browser.startup.homepage_override.mstone": "54.0.1",
    "browser.tabs.remote.autostart.2": true,
    "browser.urlbar.suggest.searches": true,
    "browser.urlbar.lastSuggestionsPromptDate": 20170522,
    "browser.urlbar.daysBeforeHidingSuggestionsPrompt": 3,
    "browser.urlbar.searchSuggestionsChoice": true,
    "browser.urlbar.userMadeSearchSuggestionsChoice": true,
    "dom.push.userAgentID": "ead0ae296d694aa29fc99c1fca161ef3",
    "extensions.lastAppVersion": "54.0.1",
    "font.internaluseonly.changed": true,
    "media.gmp-manager.buildID": "20170725113540",
    "media.gmp.storage.version.observed": 1,
    "media.webrtc.debug.log_file": "/tmp/WebRTC.log",
    "media.gmp-gmpopenh264.lastUpdate": 1495553087,
    "media.gmp-gmpopenh264.version": "1.6",
    "media.gmp-gmpopenh264.abi": "x86_64-gcc3",
    "media.gmp-manager.lastCheck": 1502449300,
    "network.cookie.prefsMigrated": true,
    "network.predictor.cleaned-up": true,
    "places.history.expiration.transient_current_max_pages": 116694,
    "places.database.lastMaintenance": 1502275645,
    "plugin.disable_full_page_plugin_for_types": "application/pdf",
    "print.print_to_file": false,
    "print.print_orientation": 0,
    "print.print_paper_name": "iso_a4",
    "print.print_unwriteable_margin_right": 25,
    "print.print_paper_height": " 11.69",
    "print.print_scaling": "  1.00",
    "print.print_unwriteable_margin_left": 25,
    "print.print_unwriteable_margin_bottom": 56,
    "print.print_duplex": 0,
    "print.print_margin_right": "0.5",
    "print.print_oddpages": true,
    "print.print_bgcolor": false,
    "print.print_paper_width": "  8.27",
    "print.print_bgimages": false,
    "print.print_shrink_to_fit": true,
    "print.print_margin_left": "0.5",
    "print.print_margin_top": "0.5",
    "print.print_unwriteable_margin_top": 25,
    "print.print_margin_bottom": "0.5",
    "print.print_in_color": true,
    "print.print_page_delay": 50,
    "print.print_paper_data": 0,
    "print.print_paper_size_unit": 0,
    "print.print_evenpages": true,
    "privacy.usercontext.about_newtab_segregation.enabled": true,
    "privacy.userContext.longPressBehavior": 2,
    "privacy.userContext.enabled": true,
    "services.sync.declinedEngines": "history,forms,tabs",
    "services.sync.lastPing": 1502705305,
    "services.sync.lastSync": "Mon Aug 14 2017 19:42:10 GMT+0100 (IST)",
    "services.sync.numClients": 4,
    "services.sync.engine.tabs": false,
    "services.sync.engine.prefs": false,
    "services.sync.engine.addons": false,
    "services.sync.engine.prefs.modified": false,
    "services.sync.engine.history": false,
    "storage.vacuum.last.places.sqlite": 1500973698,
    "storage.vacuum.last.index": 1
  },
  "lockedPreferences": {},
  "javaScript": {
    "incrementalGCEnabled": true
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.14",
      "version": "4.15"
    },
    "NSS": {
      "minVersion": "3.30.2",
      "version": "3.31"
    },
    "NSSUTIL": {
      "minVersion": "3.30.2",
      "version": "3.31"
    },
    "NSSSSL": {
      "minVersion": "3.30.2",
      "version": "3.31"
    },
    "NSSSMIME": {
      "minVersion": "3.30.2",
      "version": "3.31"
    }
  },
  "userJS": {
    "exists": false
  },
  "crashes": {
    "submitted": [],
    "pending": 0
  },
  "sandbox": {
    "hasSeccompBPF": true,
    "hasSeccompTSync": true,
    "hasPrivilegedUserNamespaces": true,
    "hasUserNamespaces": true,
    "canSandboxContent": true,
    "canSandboxMedia": true,
    "syscallLog": [],
    "contentSandboxLevel": 2
  },
  "experiments": [],
  "extensions": [
    {
      "name": "Containers Experiment",
      "version": "2.4.1",
      "isActive": true,
      "id": "@testpilot-containers"
    },
    {
      "name": "Firefox Notes",
      "version": "1.6.3dev",
      "isActive": true,
      "id": "notes@mozilla.com"
    },
    {
      "name": "Firefox Screenshots",
      "version": "6.6.0",
      "isActive": true,
      "id": "screenshots@mozilla.org"
    },
    {
      "name": "GNOME Shell integration",
      "version": "8.2.1",
      "isActive": true,
      "id": "chrome-gnome-shell@gnome.org"
    },
    {
      "name": "Min Vid",
      "version": "0.3.9",
      "isActive": true,
      "id": "@min-vid"
    },
    {
      "name": "Multi-process staged rollout",
      "version": "1.50",
      "isActive": true,
      "id": "e10srollout@mozilla.org"
    },
    {
      "name": "Page Shot",
      "version": "5.2.201701261751",
      "isActive": true,
      "id": "jid1-NeEaf3sAHdKHPA@jetpack"
    },
    {
      "name": "Pocket",
      "version": "1.0.5",
      "isActive": true,
      "id": "firefox@getpocket.com"
    },
    {
      "name": "Roomy Bookmarks Toolbar",
      "version": "1.4.14",
      "isActive": true,
      "id": "ALone-live@ya.ru"
    },
    {
      "name": "snoozetabs",
      "version": "1.0.17",
      "isActive": true,
      "id": "snoozetabs@mozilla.com"
    },
    {
      "name": "Test Pilot",
      "version": "2.0.2-dev-0d66c35",
      "isActive": true,
      "id": "@testpilot-addon"
    },
    {
      "name": "Web Compat",
      "version": "1.1",
      "isActive": true,
      "id": "webcompat@mozilla.org"
    },
    {
      "name": "Add-on Compatibility Reporter",
      "version": "2.2.4",
      "isActive": false,
      "id": "compatibility@addons.mozilla.org"
    },
    {
      "name": "HTitle",
      "version": "3.4.1-signed.1-signed",
      "isActive": false,
      "id": "{c6448328-31f7-4b12-a2e0-5c39d0290307}"
    },
    {
      "name": "Tab Center",
      "version": "1.36.0",
      "isActive": false,
      "id": "tabcentertest1@mozilla.com"
    }
  ],
  "graphics": {
    "numTotalWindows": 1,
    "numAcceleratedWindows": 0,
    "windowLayerManagerType": "Basic",
    "windowLayerManagerRemote": true,
    "currentAudioBackend": "pulse",
    "numAcceleratedWindowsMessage": [
      ""
    ],
    "adapterDescription": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Iris Plus Graphics 640 (Kaby Lake GT3) ",
    "adapterVendorID": "Intel Open Source Technology Center",
    "adapterDeviceID": "Mesa DRI Intel(R) Iris Plus Graphics 640 (Kaby Lake GT3) ",
    "adapterRAM": "",
    "adapterDrivers": "",
    "driverVersion": "3.0 Mesa 17.1.5",
    "driverDate": "",
    "webgl1Renderer": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Iris Plus Graphics 640 (Kaby Lake GT3) ",
    "webgl1Version": "3.0 Mesa 17.1.5",
    "webgl1DriverExtensions": "GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_polygon_offset GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_3DFX_texture_compression_FXT1 GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_filter_anisotropic GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_depth_clamp GL_APPLE_packed_pixels GL_APPLE_vertex_array_object GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_packed_depth_stencil GL_APPLE_object_purgeable GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_texture_array GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_ARB_ES2_compatibility GL_ARB_blend_func_extended GL_ARB_debug_output GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_stencil_export GL_ARB_shader_texture_lod GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_multisample GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_uniform_buffer_object GL_ARB_vertex_type_2_10_10_10_rev GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_NV_texture_barrier GL_ARB_get_program_binary GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_shader_precision GL_ARB_texture_compression_bptc GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shader_atomic_counters GL_ARB_shader_image_load_store GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_texture_storage GL_ARB_transform_feedback_instanced GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_transform_feedback GL_AMD_shader_trinary_minmax GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_compute_shader GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_framebuffer_no_attachments GL_ARB_invalidate_subdata GL_ARB_program_interface_query GL_ARB_robust_buffer_access_behavior GL_ARB_shader_image_size GL_ARB_shader_storage_buffer_object GL_ARB_stencil_texturing GL_ARB_texture_query_levels GL_ARB_texture_storage_multisample GL_ARB_texture_view GL_ARB_vertex_attrib_binding GL_KHR_debug GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_query_buffer_object GL_ARB_seamless_cubemap_per_texture GL_ARB_shader_draw_parameters GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_stencil8 GL_ARB_vertex_type_10f_11f_11f_rev GL_EXT_shader_integer_mix GL_INTEL_performance_query GL_ARB_clip_control GL_ARB_conditional_render_inverted GL_ARB_cull_distance GL_ARB_derivative_control GL_ARB_get_texture_sub_image GL_ARB_pipeline_statistics_query GL_ARB_shader_texture_image_samples GL_ARB_texture_barrier GL_ARB_transform_feedback_overflow_query GL_EXT_polygon_offset_clamp GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_context_flush_control GL_KHR_robust_buffer_access_behavior GL_ARB_shader_atomic_counter_ops GL_ARB_shader_clock GL_EXT_shader_samples_identical GL_KHR_texture_compression_astc_sliced_3d GL_MESA_shader_integer_functions ",
    "webgl1Extensions": "ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_frag_depth EXT_sRGB EXT_shader_texture_lod EXT_texture_filter_anisotropic EXT_disjoint_timer_query MOZ_debug_get OES_element_index_uint OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_astc WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_compressed_texture_s3tc MOZ_WEBGL_depth_texture",
    "webgl1WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_swap_method GLX_OML_sync_control GLX_SGI_make_current_read GLX_SGI_video_sync GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap GLX_EXT_buffer_age ",
    "webgl2Renderer": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Iris Plus Graphics 640 (Kaby Lake GT3) ",
    "webgl2Version": "4.5 (Core Profile) Mesa 17.1.5",
    "webgl2DriverExtensions": "GL_3DFX_texture_compression_FXT1 GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_stencil_export GL_AMD_shader_trinary_minmax GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_APPLE_object_purgeable GL_ARB_ES2_compatibility GL_ARB_ES3_1_compatibility GL_ARB_ES3_2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gpu_shader5 GL_ARB_gpu_shader_fp64 GL_ARB_gpu_shader_int64 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_post_depth_coverage GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_query_buffer_object GL_ARB_robust_buffer_access_behavior GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_precision GL_ARB_shader_stencil_export GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_stencil_texturing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_stencil8 GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_transform_feedback_overflow_query GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_64bit GL_ARB_vertex_attrib_binding GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_texture_float GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_sRGB GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_sRGB GL_EXT_texture_sRGB_decode GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_IBM_multimode_draw_arrays GL_INTEL_conservative_rasterization GL_INTEL_performance_query GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent GL_KHR_context_flush_control GL_KHR_debug GL_KHR_robust_buffer_access_behavior GL_KHR_robustness GL_KHR_texture_compression_astc_ldr GL_KHR_texture_compression_astc_sliced_3d GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_NV_conditional_render GL_NV_depth_clamp GL_NV_packed_depth_stencil GL_NV_texture_barrier GL_OES_EGL_image GL_S3_s3tc",
    "webgl2Extensions": "EXT_color_buffer_float EXT_texture_filter_anisotropic EXT_disjoint_timer_query MOZ_debug_get OES_texture_float_linear WEBGL_compressed_texture_astc WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_compressed_texture_s3tc",
    "webgl2WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_swap_method GLX_OML_sync_control GLX_SGI_make_current_read GLX_SGI_video_sync GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap GLX_EXT_buffer_age ",
    "info": {
      "AzureCanvasBackend": "skia",
      "AzureFallbackCanvasBackend": "none",
      "AzureContentBackend": "skia",
      "AzureCanvasAccelerated": 0,
      "CairoUseXRender": 0,
      "ApzWheelInput": 1,
      "ApzTouchInput": 1
    },
    "featureLog": {
      "features": [
        {
          "name": "HW_COMPOSITING",
          "description": "Compositing",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "blocked",
              "message": "Acceleration blocked by platform"
            }
          ]
        },
        {
          "name": "OPENGL_COMPOSITING",
          "description": "OpenGL Compositing",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "unavailable",
              "message": "Hardware compositing is disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER",
          "description": "WebRender",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "Build doesn't include WebRender"
            }
          ]
        }
      ],
      "fallbacks": []
    },
    "crashGuards": []
  }
}

**Comment by ksesong** [comment]
The same thing happened to me, from a slightly different situation, but the same symptoms.

Supposing I have one container named "Twitter" which is linked to *twitter.com*, with the *Always open with*. 
So, I am on a page with the "Default" container, and I left-click on a link leading to *twitter.com/xxx*: the current tab does not load the *twitter.com/xxx* page (intended behaviour), and two identical tabs are spawned instead, both with the "Twitter" container and the *twitter.com/xxx* page.
It does not manage to switch containers from the "Default" to the "Twitter" one (enforced by the URL), and creates new (2) tabs instead. There is no issue if I right-click and open the link in a new tab.

**Comment by mzavattaro** [comment]
Same issue here with youtube.com when signed in. Any timeline for a fix?

**Comment by groovecoder** [comment]
No timeline right now. I'll add the "help wanted" label to encourage contributors to look into this one.

**Comment by ShivangiKakkar** [comment]
@ph0enix49 @mzavattaro I think this issue is no more reproducible. 

@ksesong I tried to reproduce your situation: when I left-clicked  _twitter.com/xxx_ page, it was opened in a new window in the twitter container. 
>  I left-click on a link leading to twitter.com/xxx: the current tab does not load the twitter.com/xxx page (intended behaviour)

- The current tab didn't load the _twitter.com/xxx_ . 

>It does not manage to switch containers from the "Default" to the "Twitter" one (enforced by the URL), and creates new (2) tabs instead.

- Yes, it doesn't switch the containers in the same window but it opens it in a new window within the expected container.
- The issue which leads to open identical two tabs is resolved #1120 here.

**Comment by ShivangiKakkar** [comment]
@ksesong I tried to reproduce your situation: when I left-clicked  _twitter.com/xxx_ page, it was opened in a new window in the twitter container. 
>  I left-click on a link leading to twitter.com/xxx: the current tab does not load the twitter.com/xxx page (intended behaviour)

- The current tab didn't load the _twitter.com/xxx_ . 

>It does not manage to switch containers from the "Default" to the "Twitter" one (enforced by the URL), and creates new (2) tabs instead.

- It does switch the containers and opens the tab in the expected container.

I think your situation is no more reproducible. Can you please check, If so then I can close this issue. Thanks!

@ph0enix49 @mzavattaro the issue which leads to open identical two tabs is resolved #1120 here.

**Comment by ksesong** [comment]
@ShivangiKakkar I confirm that the issue I described is now fixed.

**Comment by ph0enix49** [comment]
I confirm that the issue is no longer reproduceable. Closing this.

**Comment by ShivangiKakkar** [comment]
@ksesong @ph0enix49 Thanks!!

**Comment by ARTiL1S** [comment]
> 
> 
> I confirm that the issue is no longer reproduceable. Closing this.

Well some two years later I am getting this annoying bug

**Comment by Popinaodude** [comment]
> > I confirm that the issue is no longer reproduceable. Closing this.
> 
> Well some two years later I am getting this annoying bug

So am I

**Comment by ARTiL1S** [comment]
> 
> 
> > > I confirm that the issue is no longer reproduceable. Closing this.
> > 
> > 
> > Well some two years later I am getting this annoying bug
> 
> So am I

weird thing is mine goes away on its own and then once in a while it starts to happen again

**Comment by russdb** [comment]
Definitely still happening..

**Comment by gamesguru** [comment]
I get something similar when opening a Google container into another container. Opens two copies, and neither has the back button available, have to close one and back up with the history button.  Quite annoying

**Comment by grouvig** [comment]
Duplicate of #1617
